### PR TITLE
Remove 27-character minimum threshold to use `--dmg-title`

### PR DESCRIPTION
**Pull request by sartbimpson**
While reading https://github.com/sindresorhus/create-dmg/pull/40, the 27-character min threshold has reason but the option to manually override it is sometimes necessary (for instance I'm using create-dmg to package a Hammerspoon fork and we want to keep as few changes as possible to ensure we can merge in new changes without much difficulty)

This makes the tool _slightly less_ opinionated

### CODE REVIEW

**Comment by sindresorhus** [comment]
There should at least be a note in the help output about why it should not be longer than 27.

**Comment by sartbimpson** [comment]
@sindresorhus, I didn't really get what you mean so I added a check instead

**Comment by sartbimpson** [comment]
@sindresorhus, changes made as requested

File: cli.js [hunk 1]
```diff
@@ -79,9 +79,14 @@ async function init() {
 	}
 
 	const appName = appInfo.CFBundleDisplayName || appInfo.CFBundleName;
-	const dmgTitle = appName.length > 27 ? (cli.flags.dmgTitle || appName) : appName;
+	const dmgTitle = cli.flags.dmgTitle || appName;
 	const dmgPath = path.join(destinationPath, `${appName} ${appInfo.CFBundleShortVersionString}.dmg`);
 
+	if (cli.flags.dmgTitle > 27) {
+		console.error('Disk image title exceeds 27 characters, aborting');
```
Comment (sindresorhus) [inline cli.js:86]:
```suggestion
		ora.fail('Disk image title exceeds 27 characters, aborting');
```

And I think the text should explain why more than 27 characters are not allowed.

**Comment by sartbimpson** [comment]
Added reference to original pull request

File: cli.js [hunk 2]
```diff
@@ -79,9 +79,14 @@ async function init() {
 	}
 
 	const appName = appInfo.CFBundleDisplayName || appInfo.CFBundleName;
-	const dmgTitle = appName.length > 27 ? (cli.flags.dmgTitle || appName) : appName;
+	const dmgTitle = cli.flags.dmgTitle || appName;
 	const dmgPath = path.join(destinationPath, `${appName} ${appInfo.CFBundleShortVersionString}.dmg`);
 
+	if (cli.flags.dmgTitle > 27) {
+		ora.fail('Disk image title exceeds 27 characters, aborting. For more information, see https://github.com/sindresorhus/create-dmg/pull/40');
```
Comment (sindresorhus) [inline cli.js:86]:
```suggestion
		ora.fail('The disk image title cannot exceed 27 characters. This is a limitation in a dependency: https://github.com/LinusU/node-alias/issues/7');
```

File: readme.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ $ create-dmg --help
   Options
     --overwrite          Overwrite existing DMG with the same name
     --identity=<value>   Manually set code signing identity (automatic by default)
-    --dmg-title=<value>  Manually set title of DMG volume (only used if app name is >27 character limit)
+    --dmg-title=<value>  Manually set title of DMG volume (must be less than 27 characters)
```
Comment (sindresorhus) [inline readme.md:30]: The limit is actually 27, not 26.

File: cli.js [hunk 4]
```diff
@@ -22,7 +22,7 @@ const cli = meow(`
 	Options
 	  --overwrite          Overwrite existing DMG with the same name
 	  --identity=<value>   Manually set code signing identity (automatic by default)
-	  --dmg-title=<value>  Manually set title of DMG volume (only used if app name is >27 character limit)
+	  --dmg-title=<value>  Manually set title of DMG volume (must be less than 27 characters)
```
Comment (sindresorhus) [inline cli.js:25]: I think it should say that it defaults to the app name.

**Pull request merged**